styles.css

```diff
@@ -1,5 +1,3 @@
-@import url('https://fonts.googleapis.com/css?family=Poppins:400,700,900');
-
 * {
```

Comment:
- [OPTIONAL] To make the user experience more pleasant, I'll suggest you add a smooth scroll effect so that when a user clicks on a menu item, it actually scrolls smoothly to that section. You can achieve this by adding this to the top of your CSS code. `html {
  scroll-behavior: smooth;
}`. :+1: :smile: